elijah/app.ts

```diff
@@ -0,0 +1,137 @@
+import fs from "fs";
+import readline from "node:readline/promises";
+import util from "util";
+
+const readFile = util.promisify(fs.readFile);
+
+type User = {
+  username: string;
+  displayName: string;
+  state: string;
+  friends: string[];
+};
+
+type Post = {
+  postId: string;
+  userId: string;
+  visibility: "public" | "friend";
+};
+
+let posts: Post[] = [];
+let users: User[] = [];
```

Comment:
Good way to keep track of all different Posts and Users created. Had trouble with this in my code, will implement a similar system of User and Post arrays into mine.